### ISSUE TITLE
panos_commit cant commit to Panorama

### ISSUE THREAD

**Issue by kquilliam** [PaloAltoNetworks/ansible-pan#235]
When performing a commit to Panorama using `panos_commit` without specifying a devicegroup, it is inserting a value of "None" causing the task to fail. I am just attempting to commit to Panorama w/o any device group interaction.

Playbook
```
---
- hosts: localhost
  connection: local
  roles:
   - role: PaloAltoNetworks.paloaltonetworks

  tasks:
    - include_role:
        name: pan
        tasks_from: panorama_commit
```
Play
```
- panos_commit:
    ip_address: "{{ panorama }}"
    username: "{{ user }}"
    password: "{{ password }}"
```

Output
```
fatal: [localhost]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "api_key": null,
            "devicegroup": null,
            "ip_address": "10.0.0.12",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "username": "kquilliam"
        }
    },
    "msg": "'None' device group not found in Panorama. Is the name correct?"
```

**Comment by shinmog** [comment]
OK, so doing a commit all is sending `<commit-all />` with a bunch of params.  If I try sending just that without specifying the device group, Panorama doesn't seem to like that.  This either means that commit-all requires a device group be specified or that there are additional options that need to be sent when not specifying the device group.

**Comment by kquilliam** [comment]
It just needs to send <commit/> when no device group is specified.

**Comment by shinmog** [comment]
Fixed by #338
